packages/server/modules/backgroundjobs/domain.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from 'zod'
+export const BackgroundJobStatus = {
+  Queued: 'queued',
+  // Processing: 'processing',
```

Comment:
i'm confused why is there no processing status? what happens to jobs while they're processing? surely they don't just disappear from the table

Reply:
@iainsproat can you query for all that are locked tho? i still see value in a clearly visible processing status